wifiphisher/common/interfaces.py

```diff
@@ -816,10 +816,17 @@ def get_perfect_card(phy_map_vifs, vif_score_tups):
     # sort with score
     vif_score_tuples = sorted(vif_score_tuples, key=lambda tup: -tup[1])
 
-    perfect_card, is_single_perfect_phy = get_perfect_card(
-        phy_to_vifs, vif_score_tuples)
-
-    return perfect_card, is_single_perfect_phy
+    if args.interface:
+        card = pyw.getcard(args.interface)
+        phy_number = card.phy
+        if phy_to_vifs[card.phy][0][1] == 2:
+            perfect_card = card
+            use_one_phy = True
```

Comment:
If user specified a interface which doesn't support two virtual interfaces, the `use_one_phy` variable is not defined, right?

Reply:
yes, it needs to be defined before the if condition.